.gitignore

```diff
@@ -6,3 +6,4 @@ build
 .eggs
 
 /test/e2e/actual_output
+test/java/src/main/java/com/test
```

Comment:
Please move the test output to output to `<repo>/tmp`.
On my system for a clean repo `.gradle` is also required here see:
https://github.com/juliaogris/sysl/blob/json_serializer_mv/.gitignore